src/index.ts

```diff
@@ -200,6 +208,68 @@ class SvgChunkWebpackPlugin {
 		}
 	}
 
+	/**
+	 * injects the sprite map related to the entry in the related html generated by html-webpack-plugin
+	 * @param {} compilation
+	 * @param {Array<Sprite>} sprites
+	 * @returns {Promise<void>}
+	 */
+	async handleHtmlWebpackPlugin(compilation: Compilation, sprites: Array<Sprite>): Promise<void> {
+		let HtmlWebpackPluginImport: any;
+		try {
+			// @ts-ignore
+			HtmlWebpackPluginImport = await import('html-webpack-plugin');
```

Comment:
The import is performed each time the function is called. Could the import be done upstream?

cc/ @alexander-akait

Reply:
Can we imagine a function that returns the instance and stores it as a class property so that it can be imported just once?

I think @alexander-akait has already given me feedback on the performance issue of storing this type of data as a class property

Reply:
I get the instance from the webpack compiler options